airflow/kubernetes/pod_generator.py

```diff
@@ -62,75 +68,53 @@ class PodGenerator:
     Contains Kubernetes Airflow Worker configuration logic
 
     Represents a kubernetes pod and manages execution of a single pod.
-    :param image: The docker image
```

Comment:
Why have you removed the docs?

Reply:
I found the docstring on the parameter level to be frequently incomplete and/or inaccurate. The `:type` aspect in particular often conflicts with the types that are guaranteed to be correct in the type annotations. It creates two places where code needs to be updated, but with no static check to actually guarantee that it has been done. 

I realize that the standard is having the docstring for every parameter so I will add it back, but I do feel that in some cases this is creating more maintenance cost and bloat than value being added.

Reply:
The issue is that the type annotations do not (currently) show up in the docs, so we need to  :(

As part of a separate PR https://pypi.org/project/sphinx-autodoc-typehints/ might help